table/block_based/index_builder.cc

```diff
@@ -104,6 +104,9 @@ void PartitionedIndexBuilder::MakeNewSubIndexBuilder() {
       comparator_, table_opt_.index_block_restart_interval,
       table_opt_.format_version, use_value_delta_encoding_,
       table_opt_.index_shortening, /* include_first_key */ false);
+  if (seperator_is_key_plus_seq_) {
+    sub_index_builder_->seperator_is_key_plus_seq_ = false;
```

Comment:
Why do we flip this back to false after `seperator_is_key_plus_seq_` becomes true?

Reply:
I was checking for something. Kindly ignore this change. It was in draft state initially. I will update it for review once i complete the changes.